pkg/controller/backup_controller.go

```diff
@@ -685,6 +685,11 @@ func recordBackupMetrics(log logrus.FieldLogger, backup *velerov1api.Backup, bac
 	serverMetrics.RegisterVolumeSnapshotAttempts(backupScheduleName, backup.Status.VolumeSnapshotsAttempted)
 	serverMetrics.RegisterVolumeSnapshotSuccesses(backupScheduleName, backup.Status.VolumeSnapshotsCompleted)
 	serverMetrics.RegisterVolumeSnapshotFailures(backupScheduleName, backup.Status.VolumeSnapshotsAttempted-backup.Status.VolumeSnapshotsCompleted)
+	if backup.Status.Progress != nil {
+		serverMetrics.RegisterBackupItemsBackedUpGauge(backupScheduleName, backup.Status.Progress.ItemsBackedUp)
+		serverMetrics.RegisterBackupItemsTotalGauge(backupScheduleName, backup.Status.Progress.TotalItems)
```

Comment:
We only record metrics once at the end of a backup, so my understanding is that, at the point of registering this value, `ItemsBackedUp` and `TotalItems` will be equal.

Reply:
`TotalItems` can differ from `ItemsBackedUp` according to the struct. So I think both metrics should be exposed.

https://github.com/vmware-tanzu/velero/blob/ae6e1df9aa91816dcd5d055a0325e1d54a37d3d5/pkg/apis/velero/v1/backup.go#L315-L328

Reply:
Yes, those values will be different while the backup is in progress (that is how progress information is captured and stored on the Backup CR), however, we set these are set to the same value at the end of the backup (see https://github.com/vmware-tanzu/velero/blob/main/pkg/backup/backup.go#L413-L416). As we record metrics at the end of the backup, they'll be the same.

Reply:
Ah okay, I'll remove the `backup_items_backed_up` then. Thanks for clarification 🙂

Reply:
@zubron I removed the `backup_items_backed_up` gauge. Can you PTAL?